src/main/java/uk/gov/digital/ho/hocs/info/api/EntityService.java

```diff
@@ -43,6 +43,18 @@ public List<Entity> getByEntityListName(String listName) {
         }
     }
 
+    public List<Entity> getByEntityListNameAndActive(String listName, Boolean activeOnly) {
+        if (listName != null) {
+            if (activeOnly) {
+                return entityRepository.findByEntityListSimpleName(listName);
+            } else {
+                return entityRepository.findAllByEntityListSimpleName(listName);
+            }
+        } else {
+            throw new ApplicationExceptions.EntityNotFoundException("listName was null!");
```

Comment:
Might be worth adding a specific exception, `EntityListNotFoundException` perhaps?